CONTRIBUTE.md

```diff
@@ -64,6 +64,7 @@ deploy, and therefore we try to avoid it.
 * Add **unit tests** to verify that your code not only works but also keeps working over time. When adding tests, keep
 the same directory structure used for regular classes. Try to cover **all your code** with tests. The bigger the test
 coverage, the more reliable and better our library is.
+    * `phpunit --configuration tools/phpunit tests/path/to/your/test.php`
```

Comment:
I added more info.  My goal was for someone not well versed in php to be able to create/run tests.